dotnet-desktop-guide/framework/winforms/controls/how-to-opt-out-of-file-dialog-box-automatic-upgrade.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "How To: Opt Out of File Dialog Box Automatic Upgrade"
+description: Learn how to opt out of a file dialog box automatic upgrade in Windows Forms in this 1-step procedure.
```

Comment:
I think this needs punctuation...

"Learn how to opt out of a file dialog box, automatic upgrade in Windows Forms in this 1-step procedure."

Reply:
Addressed.